cips/cip-6-second-blocks.md

```diff
@@ -0,0 +1,60 @@
+| cip | TBD |
+| - | - |
+| title | Reduce block time to six seconds |
+| description | Proposal to reduce block time on celestia-app to 6 seconds, from 12. |
+| author | Josh Stein ([@jcstein](https://github.com/jcstein)) |
+| discussions-to | <https://forum.celestia.org/t/cip-decrease-block-time-to-6-seconds/1836> |
+| status | Draft |
+| type | Standards Track |
+| category | Core |
+| created | 2024-10-09 |
+
+## Abstract
+
+This CIP proposes to reduce the block time on celestia-app to 6 seconds, from 12. This will increase the throughput by double and and reduce the time it takes for transactions to be finalized by half.
+
+## Motivation
+
+The motivation for this CIP stems from a discussion in Core Devs Call 17, where it was proposed to reduce the block time to 6 seconds from 12 seconds.
+
+## Specification
+
+1. The block time in celestia-app MUST be reduced from 12 seconds to 6 seconds.
```

Comment:
[nit] celestia-app (and celestia-core) can't guarantee block times. Currently it targets ~12 second blocks but makes blocks every ~11.77 seconds on average. Since we can't guarantee block times, perhaps this can be rephrased to say something like:

1. The block time in celestia-app should be reduced from 12 seconds to 6 seconds. Concretely this implies decreasing TimeoutCommit to _ seconds and TimeoutPropose to _ seconds.

Reply:
got it, thank you! https://github.com/celestiaorg/celestia-app/blob/79595cdac2bba302d09466485b65339431643763/pkg/appconsts/v3/app_consts.go#L11-L12 is the right place to reference here, right?

Reply:
Yea seems right